Operators/nested_ternary_operator.c

```diff
@@ -0,0 +1,38 @@
+#include <stdio.h>
+int main()
+{
+    int var= 75;
```

Comment:
int var =75, var2 =56, num; // replace  line 4-6 by this.

That's a usecase for comma operator.

Reply:
Made the changes sir, should I merge the pull request now?

Reply:
I assigned Ritika as a reviewer - let her review and merge. 
General practice: once should avoid merging his/her own pull request.
You can from your end as well, assign one of the other contributors as a reviewer from next time. Doing this just to put in place the process. Feel free to merge any other PR that you feel is okay to merge.

Reply:
Okay Sir.

Reply:
Sir, I have seen the changes and will merge the pull request now.
Also, I really liked the explanation that Anjali has provided.